config.go

```diff
@@ -133,7 +149,7 @@ func (c *config) Discover() error {
 		}
 	}
 
-	// Next, look in the plugins directory.
+	// Next, look in the default plugins directory inside the configdir/.packer.d/plugins.
```

Comment:
question: In the case of preference is there a risk that some plugin (living in both locations: default directory and `PACKER_PLUGIN_PATH`) will get stomped out since the default directory plugins get loaded after?

Reply:
Oh actually this is a good point. This should be last, not first, in order to make sure it gets precedence.